app/addons/documents/routes-index-editor.js

```diff
@@ -59,15 +60,12 @@ const IndexEditorAndResults = BaseRoute.extend({
       designDocId: '_design/' + ddoc
     });
 
-    const selectedNavItem = {
-      navItem: 'designDoc',
-      params: {
-        designDocName: ddoc,
-        designDocSection: 'Views',
-        indexName: viewName
-      }
-    };
-    SidebarActions.selectNavItem(selectedNavItem.navItem, selectedNavItem.params);
+    const selectedNavItem = new SidebarItemSelection('designDoc', {
+      designDocName: ddoc,
+      designDocSection: 'Views',
+      indexName: viewName
+    });
+    FauxtonAPI.reduxDispatch(SidebarActions.expandSelectedItem(selectedNavItem));
```

Comment:
Maybe we should wrap the `reduxDispatch` inside the `SidebarActions.expandSelectedItem` externally we don't care that its using redux.